README.md

```diff
@@ -28,7 +28,7 @@ cd llvm-project
 mkdir build
 cd build
 cmake -DLLVM_ENABLE_PROJECTS='clang;compiler-rt' -G "Unix Makefiles" ../llvm
-make -j 100  # This step is very slow
+make -j 10  # This step is very slow, be careful with this number, because increasing it too much can freeze your computer
```

Comment:
I'd say remove the comment change - it may be too distracting. (I don't want people to have to think about what number to pick, this is a complex-enough process already).

Reply:
Done!